operator/pkg/controllers/master/kcmdeployment.go

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
nit: kube-controller-manager.go

Reply:
Make folders if you want to split deployment/service/etc into separate files.

/controllers/master/kube-controller-manager/deployment.go